cli/lsp/tsc.rs

```diff
@@ -3993,22 +4000,16 @@ fn op_script_names(state: &mut OpState) -> Vec<String> {
     .collect()
 }
 
-#[derive(Debug, Deserialize, Serialize)]
-#[serde(rename_all = "camelCase")]
-struct ScriptVersionArgs {
-  specifier: String,
-}
-
 #[op2]
 #[string]
 fn op_script_version(
   state: &mut OpState,
-  #[serde] args: ScriptVersionArgs,
+  #[string] specifier: &str,
```

Comment:
Slight perf improvement - no need to use an object here if it's a single string.